web/app/js/components/LatencyOverview.jsx

```diff
@@ -160,7 +160,7 @@ export default class MultiLineGraph extends React.Component {
     return (
       <div className="current-latency">
         {
-          _.map(["P99", "P95", "P50"], latency => {
+          _.map(["P50", "P95", "P99"], latency => {
```

Comment:
As part of this change can we also reorder the latency columns in TabbedMetricsTable:

![image](https://user-images.githubusercontent.com/9226/35586410-a02c1aca-05af-11e8-9464-eda166298743.png)

Reply:
oh yeah! forgot about those, thanks!